### PR TITLE
Release/2.16.0 User collections

### PR DESCRIPTION
**Pull request by eronisko**
### Added
- user collections

### Changed
- Make (reproduction) notices editable in admin

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 All notable changes to this project will be documented in this file[^1].
 
 ## [Unreleased]
```
Comment (igor-kamil) [inline CHANGELOG.md:4]:
```suggestion
## [Unreleased]

## [2.16.0] - 2021-01-15
```

File: routes/web.php [hunk 2]
```diff
@@ -501,7 +505,7 @@ function()
     Route::get('authority/search', 'AuthorityController@search');
     Route::resource('authority', 'AuthorityController');
     Route::resource('sketchbook', 'SketchbookController');
-    Route::resource('slide', 'SlideController');
+    Route::resource('notices', 'NoticeController');
```
Comment (igor-kamil) [inline routes/web.php:508]:
```suggestion
    Route::resource('slide', 'SlideController');
    Route::resource('notices', 'NoticeController');
```

(I guess this was done unintentionally)

**Review by igor-kamil — COMMENTED** [review]
works nicely. just check my comments ⬇️ 
also next time it would be nice to keep `patternlib` updated with new components (and I appreciate how you keep of making them!) - I made separated task for that -> WEBUMENIA-1540 - it's not worth of blocking this release 😉 )

File: routes/web.php (hunk 2 above)
Comment (eronisko) [reply routes/web.php:508]: Great catch, thanks!

**Review by igor-kamil — APPROVED** [review]
🙌

**Pull request merged**